lxd/device/gpu_physical.go

```diff
@@ -396,21 +453,21 @@ func (d *gpuPhysical) postStop() error {
 
 // deviceNumStringToUint32 converts a device number string (major:minor) into separare major and
 // minor uint32s.
-func (d *gpuPhysical) deviceNumStringToUint32(devNum string) (uint32, uint32, error) {
+func (d *gpuPhysical) deviceNumStringToUint32(devNum string) (major uint32, minor uint32, err error) {
```

Comment:
This isn't related to the commit message and should be a separate commit that can be backported.
Please don't mix linter changes of existing code with new functionality.

Reply:
fixed